src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1029,8 +1030,8 @@ def _validate_and_get_connection_string(resource_group_name, storage_account):
         raise CLIError(error_message)
 
     keys = storage_client.storage_accounts.list_keys(resource_group_name, storage_account).keys
```

Comment:
Would it make sense consolidate [error reporting](https://github.com/Azure/azure-cli/blob/functionapp/src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py#L1019-L1026)  to code like:
```python
    for e in ['blob', 'queue', 'table']:
        if not getattr(endpoints, e, None):
              error_message = "Storage account has no '{}' endpoint. It must have table, queue, and blob endpoints all enabled".format(e)
```